packages/core/src/common/utils/safeInvokeMember.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2018 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the terms of the LICENSE file distributed with this project.
+ */
+
+import { isFunction } from "../utils";
+
+/**
+ * Safely invoke the member function with no arguments, if the object
+ * exists and the given key is indeed a function, and return its value.
+ * Otherwise, return undefined.
```

Comment:
nit: I wrap undefined in backticks to have it appear as preformatted in VS Code popovers.